packages/react-native-bridge/android/build.gradle

```diff
@@ -184,57 +184,52 @@ if (buildGutenbergMobileJSBundle) {
     def bundleName = 'index.android.bundle'
     def jsRootDir = '../../../..'
 
-    task bundleUpToDateCheck {
-        description("Checks if the inputs to the javascript bundle and the bundle itself are unchanged. \
+	task bundleUpToDateCheck {
+		description("Checks if the inputs to the javascript bundle and the bundle itself are unchanged. \
 If they are changed, the isBundleUpToDate flag is switched to false. That flag is used by other tasks.")
 
-        def isRelevantFile = { it.name.endsWithAny('.js', '.css') || it.name == 'package.json' }
-        def inputFiles = {
-            def jsFiles = []
-            def dirs = []
-            file(jsRootDir).eachDir { dir ->
-                if (dir.name != 'node_modules'
-                        && dir.name != 'bundle'
-                        && dir.name != 'gutenberg'
-                        && !dir.name.startsWith('symlinked')) {
-                    dirs << dir
-                }
-            }
-            file("$jsRootDir/gutenberg").eachDir { dir ->
-                if (dir.name != 'node_modules') {
-                    dirs << dir
-                }
-            }
-
-            dirs.forEach { dir ->
-                file(dir).eachFileRecurse {
-                    if (isRelevantFile(it)) {
-                        jsFiles << it
-                    }
-                }
-            }
-
-            def jsRootDirFiles = file(jsRootDir).listFiles().findAll {
-                isRelevantFile(it)
-            }
-            def gutenbergRootDirFiles = file("$jsRootDir/gutenberg").listFiles().findAll {
-                isRelevantFile(it)
-            }
-            return jsFiles + jsRootDirFiles + gutenbergRootDirFiles
-        }
-        inputs.files(inputFiles())
+		def dirs = [jsRootDir]
+		file(jsRootDir).eachDirRecurse { dir ->
+			def isRelevantDir = !['react-native-bridge/android/build/intermediates',
```

Comment:
I'm excluding `react-native-bridge/android/build/intermediates` here because these files were getting generated after a clean, which was throwing off the up-to-date check here since they are js files that are being generated:

```
build/intermediates/library_assets/debug/out/gutenberg-web-single-block/inject-css.js
build/intermediates/library_assets/debug/out/gutenberg-web-single-block/prevent-autosaves.js
build/intermediates/library_assets/debug/out/gutenberg-web-single-block/editor-style-overrides.css
```

My assumption was that we should be fine not checking these since they're generated files, but I'm not really sure what these are, so I wanted to call this out specifically in case it is something we shouldn't be excluding.

Reply:
Yea these are related to unsupported blocks, and I think this is correct that we don't need to watch for changes on these generated files